### PR TITLE
add a tip for dhcp lease file growth in large cluster

### PR DESCRIPTION
**Pull request by immarvin**
fix https://github.com/xcat2/xcat-core/issues/756

http://10.3.5.21/doc/advanced/hierarchy/configure_dhcp.html

### CODE REVIEW

File: docs/source/advanced/hierarchy/configure_dhcp.rst [hunk 1]
```diff
@@ -6,5 +6,25 @@ Add the relevant networks into the DHCP configuration, refer to: :ref:`Setup-dhc
 Add the defined nodes into the DHCP configuration, refer to:
 `XCAT_pLinux_Clusters/#configure-dhcp <http://localhost/fake_todo>`_
 
+In the large cluster, the size of dhcp lease file "/var/lib/dhcpd/dhcpd.leases" on the DHCP server will grow over time, when it increases to around 100MB, the DHCP server will take a long time to respond the DHCP requests from the clients, which will cause the DHCP timeout in the client's side: ::
```
Comment (whowutwut) [inline docs/source/advanced/hierarchy/configure_dhcp.rst]: Suggest "...grow over time.  At around 100MB in size, the DHCP server will take a long time to respond to DHCP requests from clients and cause DHCP timeouts"

File: docs/source/advanced/hierarchy/configure_dhcp.rst [hunk 2]
```diff
@@ -6,5 +6,25 @@ Add the relevant networks into the DHCP configuration, refer to: :ref:`Setup-dhc
...
+   Mar  2 01:59:10 c656ems2 dhcpd: DHCPACK on 9.114.39.101 to 00:0a:f7:73:7d:d0 via eth0
+   Mar  2 01:59:10 c656ems2 dhcpd: DHCPDECLINE of 9.114.39.101 from 00:0a:f7:73:7d:d0 via eth0: not found
+   Mar  2 01:59:10 c656ems2 dhcpd: DHCPDISCOVER from 00:0a:f7:73:7d:d0 via eth0
+   Mar  2 01:59:10 c656ems2 dhcpd: DHCPOFFER on 9.114.39.101 to 00:0a:f7:73:7d:d0 via eth0
+   Mar  2 01:59:10 c656ems2 dhcpd: DHCPREQUEST for 9.114.39.101 (9.114.39.157) from 00:0a:f7:73:7d:d0 via eth0
+   Mar  2 01:59:10 c656ems2 dhcpd: DHCPACK on 9.114.39.101 to 00:0a:f7:73:7d:d0 via eth0
+   Mar  2 01:59:10 c656ems2 dhcpd: DHCPDECLINE of 9.114.39.101 from 00:0a:f7:73:7d:d0 via eth0: not found
+   ...
 
+The solution is simply to restart the dhcpd service or run ``makedhcp -n``.
```
Comment (whowutwut) [inline docs/source/advanced/hierarchy/configure_dhcp.rst]: Is this really the solution that real sites do to resolve this? :)

**Comment by daniceexi** [comment]
I am OK with this change, but checking will be added to the **xcatprobe** command when checks the health of xCAT MN.

BTW, we may add a doc page for Q&A or Tips.

File: docs/source/advanced/hierarchy/configure_dhcp.rst (hunk 2 above)
Comment (daniceexi) [inline docs/source/advanced/hierarchy/configure_dhcp.rst]: makedhcp -n restarts dhcpd, the restart of dhcpd tidies the dhcpd.leases file

File: docs/source/advanced/hierarchy/configure_dhcp.rst (hunk 1 above)
Comment (daniceexi) [inline docs/source/advanced/hierarchy/configure_dhcp.rst]: @whowutwut Sorry, I opened the commit window before your comment, so I committed the pull request not considered your comment.

Comment (immarvin) [inline docs/source/advanced/hierarchy/configure_dhcp.rst]: hi @whowutwut ,created a new PR for your comment
https://github.com/xcat2/xcat-core/pull/848

Comment (whowutwut) [inline docs/source/advanced/hierarchy/configure_dhcp.rst]: Haha it's ok. No problem

**Pull request merged**